client/js/components/GameView/GameOverlays.vue

```diff
@@ -104,6 +113,26 @@ export default {
       }
       return mayCounter;
     },
+    showWaitingForOpponentToDiscardJackFromDeck() {
+      return (
+        this.resolvingSeven &&
```

Comment:
So did you say that you tried replacing `resolvingSeven` with the more specific `waitingForOpponentToPlayFromDeck` and it didn't work?
```suggestion
        this.waitingForOpponentToPlayFromDeck &&
```

Reply:
Removing the resolvingSeven check completely displays the overlay whenever two jacks are on top of the deck with no points, without the seven being played.

Reply:
Are you talking about removing it completely, or replacing it with `waitingForOpponentToPlayFromDeck`?

Reply:
My suggestion isn't to remove it, it's to check a different variable